### ISSUE TITLE
Cannot find module 'meteor/server-render'

### ISSUE THREAD

**Issue by cormip** [abecks/meteor-fast-render#14]
Updated to Meteor 1.6.1
Removed staringatlights:fast-render v2.16.5 (which worked before upgrade)

```
D:\project>meteor add staringatlights:fast-render

Changes to your project's package version selections:

chuangbo:cookie              added, version 1.1.0
staringatlights:fast-render  added, version 3.0.0
staringatlights:inject-data  added, version 2.2.0
```
Now when app starts:
```
Uncaught Error: Cannot find module 'meteor/server-render'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.resolve (internal/module.js:18:19)
    at Object.require (D:\project\.meteor\local\build\programs\server\boot.js:288:32)
    at makeInstallerOptions.fallback (packages\modules-runtime.js:651:18)
    at require (packages\modules-runtime.js:244:16)
    at ssr_helper.js (packages/staringatlights:fast-render/lib/server/ssr_helper.js:1:317)
    at fileEvaluate (packages\modules-runtime.js:343:9)
    at require (packages\modules-runtime.js:238:16)
    at D:\project\.meteor\local\build\programs\server\packages\staringatlights_fast-render.js:2172:1
    at D:\project\.meteor\local\build\programs\server\packages\staringatlights_fast-render.js:2177:3
    at infos.forEach.info (D:\project\.meteor\local\build\programs\server\boot.js:418:13)
    at Array.forEach (<anonymous>)
    at D:\project\.meteor\local\build\programs\server\boot.js:417:9
    at D:\project\.meteor\local\build\programs\server\boot.js:471:5
    at Function.run (D:\project\.meteor\local\build\programs\server\profile.js:510:12)
    at D:\project\.meteor\local\build\programs\server\boot.js:470:11
```

**Comment by abecks** [comment]
Sorry about that, the dependency on `server-render` is missing, I will push a patch for that. You can `meteor add server-render` in the meantime to fix this.

**Comment by abecks** [comment]
Fixed in 3.0.1

**Comment by cormip** [comment]
I think you may need to revisit that fix. I tried installing 3.0.1 and still got the error. 

I then directly installed ```meteor add server-render``` and still got the same "Cannot find module 'meteor/server-render'" error.
